static/js/public/store-categories.js

```diff
@@ -15,18 +36,8 @@ function getCategory(holder) {
   // Write the html and init the carousel
   const writeCategory = function(response) {
     holder.innerHTML = response;
-    new Swiper(
-      holder.querySelector(".swiper-container"),
-      Object.assign(
-        {
-          navigation: {
-            nextEl: `.swiper-button__next--${category}`,
-            prevEl: `.swiper-button__prev--${category}`
-          }
-        },
-        CATEGORY_CONFIG
-      )
-    );
+
+    getColour(holder);
```

Comment:
By looking at the coded I get am impression that colour calculations are only going to happen for categories on store page, but not on other pages (like publisher details, iot, etc).
Is `getColour` used elsewhere?

Reply:
not yet. my next task is to do the same for the category pages so I'll probably extract it somewhere then